_includes/quick-start-module.js

```diff
@@ -98,58 +100,44 @@ function getPreferredCuda(os) {
   if (os == 'macos') {
     return 'accnone';
   }
-  return 'cuda11.6';
+  return 'cuda.x';
 }
 
 // Disable compute platform not supported on OS
 function disableUnsupportedPlatforms(os) {
-  supportedComputePlatforms.forEach( (oses, platform, arr) => {
-    var element = document.getElementById(platform);
-    if (element == null) {
-      console.log("Failed to find element for platform " + platform);
+  for (const [arch_key, info] of archInfoMap) {
```

Comment:
Hmm, we don't run `babeljs` on this one, do we? What is the browser support matrix `for..of` loops? (I vaguely remember I had to roll back to `forEach` because it was not supported at a time on Safari or something like that)

Reply:
Looks like all the browsers (except  IE and Opera Mini) support es6 syntax now : https://caniuse.com/?search=es6